### ISSUE TITLE
Compatibility with opm==2021.10

### ISSUE THREAD

**Issue by berland** [equinor/ecl2df#392]
ecl2df is currently not compatible with OPM 2021.10, this failure must be solved:

```
=================================== FAILURES ===================================
__________________________ test_inject_xxxdims_ntxxx ___________________________

    def test_inject_xxxdims_ntxxx():
        """Test the wrapper of inject_dimcount()"""
    
        with pytest.raises(AssertionError):
            inferdims.inject_xxxdims_ntxxx("WELLDIMS", "NTSFUN", "SWOF\n0/\n0/\n")
        with pytest.raises(AssertionError):
            inferdims.inject_xxxdims_ntxxx("TABDIMS", "NTFUN", "SWOF\n0/\n0/\n")
    
        # Repeated calls should be ok:
        deck_with_injection = inferdims.inject_xxxdims_ntxxx(
            "TABDIMS", "NTSFUN", "SWOF\n0/\n0/\n", 1
        )
>       assert "TABDIMS\n   1 /" in str(
            inferdims.inject_xxxdims_ntxxx("TABDIMS", "NTSFUN", deck_with_injection, 10)
        )
E       AssertionError: assert 'TABDIMS\n   1 /' in 'TABDIMS\n 1 /\nSWOF\n 0 /\n'
E        +  where 'TABDIMS\n 1 /\nSWOF\n 0 /\n' = str(<opm.libopmcommon_python.Deck object at 0x7fb3b054d070>)
E        +    where <opm.libopmcommon_python.Deck object at 0x7fb3b054d070> = <function inject_xxxdims_ntxxx at 0x7fb3b1530ca0>('TABDIMS', 'NTSFUN', <opm.libopmcommon_python.Deck object at 0x7fb3b054d070>, 10)
E        +      where <function inject_xxxdims_ntxxx at 0x7fb3b1530ca0> = inferdims.inject_xxxdims_ntxxx

/home/runner/work/ecl2df/ecl2df/tests/test_inferdims.py:99: AssertionError
```

**Comment by berland** [comment]
This looks only like a whitespace change.

**Comment by asnyv** [comment]
closed by #395
